reporter/src/main/kotlin/reporters/NoticeReporter.kt

```diff
@@ -46,9 +46,13 @@ class NoticeReporter : Reporter() {
         // TODO: Decide whether we want to merge the list of detected licenses with declared licenses (which do not come
         // with a copyright).
         scanRecord.scanResults.forEach { container ->
-            container.results.forEach { result ->
-                result.summary.licenseFindings.forEach { licenseFinding ->
-                    allFindings.getOrPut(licenseFinding.license) { sortedSetOf() } += licenseFinding.copyrights
+            val excludes = ortResult.repository.config.excludes
+
+            if (ortResult.analyzer?.result?.includesPackage(container.id, excludes) == true) {
```

Comment:
I'm wondering whether we should change the API so that we could write here:

    if (!excludes.isExcluded(container.id)) {

I.e. add an `fun isExcluded(id: Identifier)` to the `Excludes` class, similar to `fun isExcluded(project: Project)`.

Reply:
I agree it's better to have this logic in the exclude related classes, I will refactor it. But the API will be different, as `isExcluded` for a package makes only sense in the context of an `AnalyzerResult`.